src/Germany/Mews.Fiscalizations.Germany/Model/Transaction.cs

```diff
@@ -26,6 +34,14 @@ public sealed class Transaction
 
         public Guid Id { get; }
 
+        public Guid ClientId { get; }
+
+        public Guid TssId { get; }
+
+        public string ClientSerialNumber { get; }
+
+        public string TssSerialNumber { get; }
```

Comment:
Is CertificateSerial useful after adding these?

Reply:
I would keep it till we confirm what actually should be displayed on the bill.

Then i can revisit this and delete anything we're not showing on the invoice.

Reply:
If the API is still providing this value, I'd definitely keep it. The lib should be usable for wider audience, so there's no reason to delete the properties just because we are not using them.

My main point was that maybe this was replaced by the new serial numbers. But if that's not the case, let's keep it the way it is.

Reply:
Nope, i don't think anything replaces anything.